pages/common/emacs.md

```diff
@@ -14,3 +14,7 @@
 - Exit emacs (save buffers and terminate):
 
 `Ctrl + X, Ctrl + C`
+
+- Open a file in emacs at a specified line number:
+
+`emacs +{{line number}} {{filename}}`
```

Comment:
```suggestion
emacs +{{line_number}} {{path/to/file}}
```

It'd be good if you could change the other `{{filename}}` to `{{path/to/file}}` as well.

Reply:
For sure!